scenario/context.py

```diff
@@ -446,13 +435,11 @@ def __init__(
         It contains: the charm source code being executed, the metadata files associated with it,
         a charm project repository root, and the juju version to be simulated.
 
-        After you have instantiated Context, typically you will call one of `run()` or
-        `run_action()` to execute the charm once, write any assertions you like on the output
-        state returned by the call, write any assertions you like on the Context attributes,
-        then discard the Context.
+        After you have instantiated Context, typically you will call `run()` to execute the charm
+        once, write any assertions you like on the output state returned by the call, write any
+        assertions you like on the Context attributes, then discard the Context.
```

Comment:
uhm, why is this duplicated (see above, line 346)?

Reply:
Ah, I've done a bad merge at some point. For the Sphinx autodoc'ing I moved most of `Context.__init__.__doc__` to `Context.__doc__` but it's ended up with both copies (probably there was a clash and I mistakenly accepted both). I'll clean that up.